src/keygen/Main.hs

```diff
@@ -1,121 +1,87 @@
-{-# LANGUAGE ApplicativeDo       #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 
 module Main where
 
+import           Data.Aeson           (eitherDecode)
 import qualified Data.ByteString.Lazy as BSL
 import qualified Data.HashMap.Strict  as HM
 import qualified Data.Text            as T
-import           Options.Applicative  (Parser, ParserInfo, auto, execParser, fullDesc,
-                                       help, helper, info, long, metavar, option, option,
-                                       progDesc, short, strOption, value)
+import           Options.Applicative  (execParser)
 import           Prelude              (show)
 import           System.Directory     (createDirectoryIfMissing)
 import           System.FilePath      (takeDirectory)
-import           System.Random        (randomRIO)
 import           Universum            hiding (show)
```

Comment:
Can you remove this `hiding (show)`? It's not needed – `show` from Universum can produce both `Text` and `String`